packages/near-membrane-dom/src/window.ts

```diff
@@ -83,7 +83,9 @@ export function filterWindowKeys(keys: PropertyKey[], remapTypedArrays: boolean)
 
     // Crypto and typed arrays must be from the same global object
     if (remapTypedArrays === false) {
+        result.splice(result.indexOf('crypto'), 1);
```

Comment:
doing 3 indexOf on a 1088 entries  Array of strings is highly inefficient. This needs to be done and faster. Maybe a Set needs to be used instead of the Array so we can have O(1) operations on lookups and deletes.